src/EnergyPlus/UnitarySystem.cc

```diff
@@ -1428,6 +1434,9 @@ namespace UnitarySystems {
 
         bool errorsFound(false);
 
+        UnitarySys::getDXCoilSystemData(state, objectName, ZoneEquipment, ZoneOAUnitNum, errorsFound);
+        // call UnitarySystem getInput last, where all report variables will be set up, even if no UnitarySystem objects exist in input.
+        // see function setupAllOutputVars to add new system specific report variables.
         UnitarySys::getUnitarySystemInputData(state, objectName, ZoneEquipment, ZoneOAUnitNum, errorsFound);
```

Comment:
New objects will each have their own getInput and read that object, translate to UnitarySystem inputs and then call processInputSpec. Object specific report variables will be processed last after memory is finalized. Where/when report variable processing occurs is up for debate.